lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -204,6 +209,11 @@ Function Choco-Upgrade
         $cmd += " -ignoredependencies"
```

Comment:
Ok their docs say otherwise https://github.com/chocolatey/choco/wiki/CommandsReference#how-to-pass-options--switches and all long form options start with `--`. I'll keep an eye on it but I suppose we would have heard from others if it didn't work.